src/component/login-form/Loginform.jsx

```diff
@@ -23,18 +28,49 @@ function LoginForm({onClickSignUp, onSubmit}) {
     };
 
     const handleChange = (e) => {
-        const {name, value} = e.target;
+        const {name, value, type, checked} = e.target;
         setFormData({
             ...formData,
-            [name]: e.target.name === 'rememberMe' ? e.target.checked : value,
+            [name]: type === 'checkbox' ? checked : value,
         });
     };
 
-    const handleSubmit = (e) => {
+    const handleSubmit = async (e) => {
+        const API_URL = config.API_URL;
         e.preventDefault();
         const validationErrors = validate();
         if (Object.keys(validationErrors).length === 0) {
-            onSubmit(formData);
+            try {
+                const response = await fetch(`${API_URL}/api/post/login`, {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify({
+                        email: formData.email,
+                        password: formData.password,
+                    }),
+                });
+    
+                const text = await response.json();
+    
+                if (response.ok) {
+                    setMessage('User logged in successfully!');
+                    if (text.data.isOrg === false) {
```

Comment:
lol, I just realized this as I was testing it on my end. It was supposed to be  if `isOrg === true` then you redirect to `org-dashboard` 😅 @Donnyd88 , No worries I am going to correct that and then push through the `eventsPage` branch.